sympy/sets/sets.py

```diff
@@ -366,6 +366,12 @@ def is_subset(self, other):
 
         """
         if isinstance(other, Set):
+            dispatch = getattr(self, '_eval_is_subset', None)
+            if dispatch is not None:
+                ret = dispatch(other)
+                if ret is not None:
+                    return ret
```

Comment:
Can we just `return dispatch(other)`?

Reply:
If it returns, then the check using the computation of the intersection may not be true.
Though generally, I would think that having to compute intersection every time and relying on the automatic canonicalization may not be a good idea for testing the subset.